src/core/utils/fs.h

```diff
@@ -59,6 +59,11 @@ namespace Utils
         bool forceRemove(const QString& file_path);
         void removeDirRecursive(const QString& dirName);
 
+#ifndef DISABLE_GUI
+        void openPath(const QString& absolutePath);
+        void openFolderSelect(const QString& absolutePath);
```

Comment:
Yeah, it went back and forth between the two. It was in Misc initially, I don't know why I later moved it here.